Resources/public/templates/fields/edit/time.hbt

```diff
@@ -9,11 +9,38 @@
         </label>
     </div>
     <div class="pure-u ez-editfield-input-area ez-default-error-style">
-        <div class="ez-editfield-input"><div class="ez-time-input-ui"><input type="time" {{#if useSeconds}}step="1"{{/if}}
-                value="{{ html5InputTime }}"
-                class="ez-validated-input"
-                id="ez-field-{{ content.contentId }}-{{ fieldDefinition.identifier }}"
-                {{#if isRequired}} required{{/if}}
-            ></div></div>
+        <div class="ez-editfield-input"><div class="ez-time-input-ui">
+            {{#if supportsTimeInput}}
+                <input type="time" {{#if useSeconds}}step="1"{{/if}}
+                     value="{{ html5InputTime }}"
+                     class="ez-validated-input"
+                     id="ez-field-{{ content.contentId }}-{{ fieldDefinition.identifier }}"
+                      {{#if isRequired}} required{{/if}}
+                >
+            {{else}}
+                <input type="text"
```

Comment:
with https://github.com/ezsystems/PlatformUIBundle/pull/149/files#r22789925 this can also be simplified. The input can always have `type="time"` even if time is not supported. When time is not supported, you'll just have to add the `pattern` and the `placeholder` (which are different depending on `useSeconds`).